tractor/_actor.py

```diff
@@ -575,6 +575,13 @@ async def _async_main(
                         await chan.connect()
                         # initial handshake, report who we are, who they are
                         await self._do_handshake(chan)
+
+                        if self._spawn_method == "trio":
+                            # recieve additional init params
+                            self._parent_main_data = await chan.recv()
```

Comment:
Maybe just send this as a single `dict`:

```python
parent_data = await chan.recv()
for attr, value in parent_data.items():
    setattr(self, attr, value)
```
This way we can change the names without having to change it all here every time?